matching-notification-migrations/src/main/java/org/phenotips/matchingnotification/internal/R71502PatientNetwork196DataMigration.java

```diff
@@ -134,7 +134,7 @@ public void hibernateMigrate() throws DataMigrationException, XWikiException
                     // re-compute the match with just fetched two local patients
                     PatientSimilarityView similarityView = this.factory.makeSimilarPatient(matchedPatient.getPatient(),
                         referencePatient.getPatient());
-                    PatientMatch newMatch = new DefaultPatientMatch(similarityView, null, null);
+                    PatientMatch newMatch = new CurrentPatientMatch(similarityView, null, null);
```

Comment:
@marta- @sdumitriu PN-401 changes `PatientMatch` class implementation and breaks `PatientMatch.class.getCanonicalName()` in other migrators. They also have to be changed to use `CurrentPatientMatch` instead, even if we think we will never run them again. Not a big change, but we will have not broken code in repo.

Reply:
Ads discussed, we will ignore older migrators for PC